src/bin/cargo/commands/clean.rs

```diff
@@ -7,7 +7,7 @@ pub fn cli() -> App {
         .about("Remove artifacts that cargo has generated in the past")
         .arg_package_spec_simple("Package to clean artifacts for")
         .arg_manifest_path()
-        .arg_target_triple("Target triple to clean output for (default all)")
+        .arg_target_triple("Target triple to clean output for")
```

Comment:
What @ehuss said ^

Perhaps:
```suggestion
        .arg_target_triple("Target triple to clean output for (defaults to all targets)")
```